crypto/src/hash/prefixes.rs

```diff
@@ -93,7 +93,28 @@ mod tests {
     }
 
     #[test]
-    fn make_merkle_tree_hash() {
+    fn make_merkle_tree_hash_3() {
```

Comment:
In `create()` above, please discontinue use of single letter variable names.

Optionally, since the prefix is always a string, another way could be

```rust
fn create(prefix: &str) -> [u8; PREFIX_BYTE_LEN] {
    let prefix = &prefix[..std::cmp::min(prefix.len(), PREFIX_BYTE_LEN)];
    format!("{:*<PREFIX_BYTE_LEN$}", prefix)
        .as_bytes()
        .try_into()
        .expect("bad length")
}
```
N.b. bad length is not possible.

Reply:
Also, could really use some rustdoc comments

Reply:
This was a const fn to calculate predefined prefixes at compile time instead of runtime, so str or string types are not feasible, will rename the locals  (this fn is not introduced in this PR tho)